packages/diagram/src/module/diagrams/content/uml/sequence/event.ts

```diff
@@ -0,0 +1,135 @@
+import { fun, id, InterpreterModule, numberType, optional, stringType } from "@hylimo/core";
+import { SCOPE } from "../../../../base/dslModule.js";
+
+/**
+ * Defines an event.<br>
+ * An event is an abstract representation of a point on the timeline.
+ */
+export const eventModule = InterpreterModule.create(
+    "uml/sequence/event",
+    ["uml/sequence/defaultValues"],
+    [],
+    [
+        id(SCOPE).assignField(
+            "event",
+            fun(
+                `
+                (name, yDistance) = args
+
+                eventObject = [name = name]
+                if(!(scope.internal.registerInDiagramScope(name, eventObject))) {
+                    error("Cannot construct event '" + name + "' as this variable is already declared somewhere else")
+                }
+
+                originalArgs = args
+
+                // The event itself must store its y-coordinate so that the calculation of the activity indicators and co works correctly
+                previousEvent = scope.internal.lastSequenceDiagramEvent
+                eventObject.deltaY = if(yDistance != null) { yDistance } { scope.eventDistance }
+                eventObject.y = if(previousEvent != null) { previousEvent.y } { 0 } + eventObject.deltaY 
+
+                // When in debugging mode, print the event name on the left to give an overview where we are
+                if(scope.enableDebugging) {
+                    start = scope.apos(-100, eventObject.y) // a little bit to the left of the diagram to leave space for the participants
```

Comment:
Technically no, but practically you would have to be really stupid to change the value.
There's **no** benefit to it, it only makes more work since multiple diagrams simultaneously are not possible.
So no, I don't think it is worth to be fixed.

Reply:
well, as it's only the debug display, I guess it's fine